recipes/pyomo/meta.yaml

```diff
@@ -0,0 +1,169 @@
+{% set version = "4.3.11377" %}
+
+package:
+  name: pyomo
+  version: {{ version }}
+
+source:
+  fn: Pyomo-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/12/8e/edae55601011b4356f26e3d3346e0032a67a89cd2621ff893e0a623d68ab/Pyomo-4.3.11377.tar.gz
+  #url: https://pypi.python.org/packages/source/P/Pyomo/Pyomo-{{ version }}.tar.gz
```

Comment:
Will do.  I left it while debugging the url.  I really liked the simpler
url, but oh well.
On Apr 27, 2016 8:10 AM, "Filipe" <notifications@github.com> wrote:

In recipes/pyomo/meta.yaml
<https://github.com/conda-forge/staged-recipes/pull/449#discussion_r61246137>
:

> @@ -0,0 +1,169 @@
> +{% set version = "4.3.11377" %}
> +
> +package:
> +  name: pyomo
> +  version: {{ version }}
> +
> +source:
> +  fn: Pyomo-{{ version }}.tar.gz
> +  url: https://pypi.python.org/packages/12/8e/edae55601011b4356f26e3d3346e0032a67a89cd2621ff893e0a623d68ab/Pyomo-4.3.11377.tar.gz
> +  #url: https://pypi.python.org/packages/source/P/Pyomo/Pyomo-{{ version }}.tar.gz

So just remove the commented URL. I am not sure why some URLs are re-direct
and others are not. This change in PyPI is really annoying.

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub
<https://github.com/conda-forge/staged-recipes/pull/449/files/ebc47763a7a7e7672134bb0b59d5f4486184a438#r61246137>